README.md

```diff
@@ -4,11 +4,11 @@ Fora is a protocol for building self-hosted, self-regulating Internet communitie
 
-A **self-regulating** community is a community in which the users themselves collectively decide on the rules and who will enforce them. These rules and their enforcers must also be changable at any time by the community. This allows communities to decide their own rules of conduct and appoint specific users to enforce these rules on behalf of the community. However, these enforcers cannot abuse their power without risking getting overthrown and replaced by the community.
+A **self-regulating** community is a community in which the members themselves collectively decide on the rules and who will enforce them. These rules and their enforcers must also be changable at any time by the community. This allows communities to decide their own rules of conduct and appoint specific members to enforce these rules on behalf of the community. However, these enforcers cannot abuse their power without risking getting overthrown and replaced by the community.
 
-Virtually all major internet platforms today do not fit the description above. Take Facebook as an example community. The Facebook community is the set of Facebook users who all understand that every Facebook user is also part of the Facebook community. The Facebook community interacts through Facebook posts, messages, comments, etc. The community rules are the Facebook content policy and Facebook, Inc is ultimately responsible for enforcing them. Thus Facebook is a community as defined above, yet it is not self-hosted since one must rely on Facebook servers to access the community. It is also not self-regulating since Facebook, Inc unilaterally sets the content policy and unilaterally enforces it, without any mechanism for Facebook users to have input on the rules or their enforcement. This is a problem since the incentives of the central operator can become grossly misaligned with the welfare of the community without an effective way for the community to respond.
+Virtually all major internet platforms today do not fit the description above. Take Facebook as an example community. The Facebook community is the set of Facebook members who all understand that every Facebook member is also part of the Facebook community. The Facebook community interacts through Facebook posts, messages, comments, etc. The community rules are the Facebook content policy and Facebook, Inc is ultimately responsible for enforcing them. Thus Facebook is a community as defined above, yet it is not self-hosted since one must rely on Facebook servers to access the community. It is also not self-regulating since Facebook, Inc unilaterally sets the content policy and unilaterally enforces it, without any mechanism for Facebook members to have input on the rules or their enforcement. This is a problem since the incentives of the central operator can become grossly misaligned with the welfare of the community without an effective way for the community to respond.
```

Comment:
```suggestion
Virtually all major internet platforms today do not fit the description above. Take Facebook as an example community. The Facebook community is the set of Facebook users who all understand that every Facebook user is also part of the Facebook community. The Facebook community interacts through Facebook posts, messages, comments, etc. The community rules are the Facebook content policy and Facebook, Inc is ultimately responsible for enforcing them. Thus Facebook is a community as defined above, yet it is not self-hosted since one must rely on Facebook servers to access the community. It is also not self-regulating since Facebook, Inc unilaterally sets the content policy and unilaterally enforces it, without any mechanism for Facebook users to have input on the rules or their enforcement. This is a problem since the incentives of the central operator can become grossly misaligned with the welfare of the community without an effective way for the community to respond.
```

Reply:
Reverted because `user` makes more sense in the Facebook terminology